views/partials/nhl_player_card

```diff
@@ -0,0 +1,41 @@
+
+<script id="card" type="text/x-handlebars-template">
```

Comment:
the only thing that will change are whatever the stats are called right? We should just do a a sub sub template in this sub template